src/snowflake/snowpark/session.py

```diff
@@ -1390,7 +1390,7 @@ def create_dataframe(
                 schema=sf_schema,
                 quote_identifiers=True,
                 auto_create_table=True,
-                create_temp_table=True,
+                table_type="temporary",
```

Comment:
also remove it from write_pandas?

Reply:
`write_pandas` is already using `table_type`. I guess this is some sort of missed usage.